### PR TITLE
No op implementation to remove stubs in remote

### PR DESCRIPTION
**Pull request by boks1971**

### CODE REVIEW

File: pkg/rtc/types/interfaces.go [hunk 1]
```diff
@@ -32,86 +32,110 @@ const (
...
 
 	// permissions
-	CanPublish() bool
-	CanSubscribe() bool
-	CanPublishData() bool
 	Hidden() bool
-	IsRecorder() bool
-	SubscriberAsPrimary() bool
 
 	Start()
```
Comment (boks1971) [inline pkg/rtc/types/interfaces.go:126]: `Start()` and `DebugInfo()` below is not currently needed in `RemoteParticipant`, but left it in there as they seem useful and also for symmetry with `Close` method.

File: pkg/rtc/types/interfaces.go [hunk 2]
```diff
@@ -32,86 +32,110 @@ const (
...
-
 	UpdateVideoLayers(updateVideoLayers *livekit.UpdateVideoLayers) error
 
-	UpdateSubscribedQuality(nodeID string, trackID livekit.TrackID, maxQuality livekit.VideoQuality) error
-
-	UpdateMediaLoss(nodeID string, trackID livekit.TrackID, fractionalLoss uint32) error
-
 	DebugInfo() map[string]interface{}
+
+	LocalParticipant
```
Comment (cnderrauber) [inline pkg/rtc/types/interfaces.go:138]: maybe better to put this at beginning of  structure

**Review by cnderrauber — APPROVED** [review]
looks good! just a comment about position of embed interface

File: pkg/rtc/types/interfaces.go (hunk 2 above)
Comment (boks1971) [reply pkg/rtc/types/interfaces.go:138]: Will do

**Pull request merged**